### PR TITLE
Sg/highlight properties from blackboard

### PR DESCRIPTION
**Pull request by martint-unity**
### Purpose of this PR
Added ability to hover properties from black board to get them highlighted in the graph and vice versa.

---
### Release Notes
Please add any useful notes about the feature/fix that might be helpful for others.

---
### Testing status
https://katana.bf.unity3d.com/projects/com.unity.render-pipelines/builders?ScriptableRenderLoop_branch=sg%2Fhighlight-properties-from-blackboard&automation-tools_branch=add-platform-filter&unity_branch=trunk

**Manual Tests**: What did you do?

**Automated Tests**: What did you setup?

Any test projects to go with this to help reviewers?

---
### Overall Product Risks
**Technical Risk**: None, Low, Medium, High?
low

**Halo Effect**: None, Low, Medium, High?
low

---
### Comments to reviewers
Notes for the reviewers you have assigned.

### CODE REVIEW

File: com.unity.shadergraph/Editor/Drawing/Views/MaterialNodeView.cs [hunk 1]
```diff
@@ -196,6 +201,32 @@ public void Initialize(AbstractMaterialNode inNode, PreviewManager previewManage
             }
         }
 
+        void OnMouseHover(EventBase evt)
+        {
+            var grahView = GetFirstAncestorOfType<GraphEditorView>();
```
Comment (pbbastian) [inline com.unity.shadergraph/Editor/Drawing/Views/MaterialNodeView.cs]: rename to `graphView`

File: com.unity.shadergraph/Editor/Drawing/Views/MaterialNodeView.cs [hunk 2]
```diff
@@ -196,6 +201,32 @@ public void Initialize(AbstractMaterialNode inNode, PreviewManager previewManage
...
+            if(grahView == null)
```
Comment (pbbastian) [inline com.unity.shadergraph/Editor/Drawing/Views/MaterialNodeView.cs]: super nitpick time: `if(` -> `if (`

File: com.unity.shadergraph/Editor/Drawing/Views/MaterialNodeView.cs [hunk 3]
```diff
@@ -196,6 +201,32 @@ public void Initialize(AbstractMaterialNode inNode, PreviewManager previewManage
...
+                return;
+
+            var blackboardProvider = grahView.blackboardProvider;
+            if(blackboardProvider == null)
```
Comment (pbbastian) [inline com.unity.shadergraph/Editor/Drawing/Views/MaterialNodeView.cs]: super nitpick time: `if(` -> `if (`

**Comment by pbbastian** [comment]
@martint-unity Can you merge in `master`?

**Comment by martint-unity** [comment]
@pbbastian 
Update: Master is merged

**Comment by martint-unity** [comment]
@pbbastian Updated with your suggestions

**Pull request merged**